packages/components/src/DataList/components/DataListItemActionsOverflow/DataListItemActionsOverflow.tsx

```diff
@@ -22,7 +22,7 @@ export function DataListItemActionsOverflow<T extends DataListObject>({
         <Button
           icon="more"
           ariaLabel="More actions"
-          type="secondary"
+          type="tertiary"
```

Comment:
tertiary button in the "more" actions

unclear to me at this point why this action is different from the [DataList actions](https://github.com/GetJobber/atlantis/pull/1915/files#r1764033275) but here we are

Reply:
It's a good separation of concerns.
`DataListActions` are rendering the primary actions we see on hover. `DataListItemActionsOverflow` is that ••• menu button and needs to handle state for opening/closing the menu

I think we could also pretty easily extract & simplify `DataListItemActionsOverflow` to share with other menus that would have the same behaviour. Obviously not something we'll do now but, hope that helps clear things a little